dashboard/app/controllers/courses_controller.rb

```diff
@@ -62,6 +62,7 @@ def show
 
     @page_title = @unit_group.localized_title
     @page_description = I18n.t("data.course.name.#{@unit_group.name}.description_short", default: '').truncate(200, separator: '.', omission: '.')
+    @canonical_url = CDO.studio_url(UnitGroup.latest_stable_version(@unit_group.family_name)&.link) if UnitGroup.latest_stable_version(@unit_group.family_name)&.link
```

Comment:
Nit: To simplify having to run `UnitGroup.latest_stable_version(@unit_group.family_name)&.link` twice back to back, could it be saved to a variable? This would also make the line a bit more readable too I think